  GNU nano 4.8                                                               i2c1.patch                                                                         
                        AM33XX_IOPAD(0x9b4, PIN_OUTPUT_PULLDOWN | MUX_MODE3)    /* xdma_event_intr1.clkout2 */
@@ -195,6 +216,27 @@
        status = "okay";
 };
 
+&uart1 {
+    pinctrl-names = "default";
+    pinctrl-0 = <&uart1_pins>;
+
+    status = "okay";
+};
+
+&uart4 {
+    pinctrl-names = "default";
+    pinctrl-0 = <&uart4_pins>;
+
+    status = "okay";
+};
+
+&uart5 {
+    pinctrl-names = "default";
+    pinctrl-0 = <&uart5_pins>;
+
+    status = "okay";
+};
+
 &usb {
        status = "okay";
 };
-- 
2.17.1
